content/concept-amendments.md

```diff
@@ -86,13 +86,139 @@ When an amendment gets enabled for the network after the voting process, servers
+
+To find out which features are blocking your `rippled` server, use the [`feature`](reference-rippled.html#feature) admin command. Look for features that have `"enabled" : true` and `"supported" : false`. These values for a feature mean that the amendment is currently enabled (required) in the latest ledger, but your server does not know how to support, or apply, the amendment.
+
+**Example JSON-RPC Response:**
+
+```
+{
+    "result": {
+        "features": {
+            "07D43DCE529B15A10827E5E04943B496762F9A88E3268269D69C44BE49E21104": {
+                "enabled": true,
+                "name": "Escrow",
+                "supported": true,
+                "vetoed": false
+            },
+            "08DE7D96082187F6E6578530258C77FAABABE4C20474BDB82F04B021F1A68647": {
+                "enabled": true,
+                "name": "PayChan",
+                "supported": true,
+                "vetoed": false
+            },
+            "1562511F573A19AE9BD103B5D6B9E01B3B46805AEC5D3C4805C902B514399146": {
+                "enabled": false,
+                "name": "CryptoConditions",
+                "supported": true,
+                "vetoed": false
+            },
+            "157D2D480E006395B76F948E3E07A45A05FE10230D88A7993C71F97AE4B1F2D1": {
+                "enabled": true,
+                "supported": false,
+                "vetoed": false
+            },
+...
+            "67A34F2CF55BFC0F93AACD5B281413176FEE195269FA6D95219A2DF738671172": {
+                "enabled": true,
+                "supported": false,
+                "vetoed": false
+            },
+...
+            "F64E1EABBE79D55B3BB82020516CEC2C582A98A6BFE20FBE9BB6A0D233418064": {
+                "enabled": true,
+                "supported": false,
+                "vetoed": false
+            }
+        },
+        "status": "success"
+    }
+}
+```
+
+In this example, conflicts with the following features are causing your `rippled` server to be amendment blocked:
+
+* `157D2D480E006395B76F948E3E07A45A05FE10230D88A7993C71F97AE4B1F2D1`
+
+* `67A34F2CF55BFC0F93AACD5B281413176FEE195269FA6D95219A2DF738671172`
+
+* `F64E1EABBE79D55B3BB82020516CEC2C582A98A6BFE20FBE9BB6A0D233418064`
+
+To look up which `rippled` version supports these features, see [Known Amendments](reference-amendments.html).
+
+***TODO: Question: Do we think this is the workflow users will use to figure this out? If yes, then it might be nice for the user if the table at the top of the Known Amendments page would list amendment IDs. I know that it makes the table a little uglier, but the feature response doesn't always provide the short name and always provides the ID. The way the page is currently structured, the user will need to ctrl+f for the ID, note the short name, and then scroll back up to the table at the top to note the version. They'll need to do this for each amendment.***
```

Comment:
Yeah, I was thinking about changing the Known Amendments table to list them like that. It quickly starts to run off the side of the page on some screens, though. That's something to address in a different task.